src/network/networkprotocol.h

```diff
@@ -610,6 +611,20 @@ enum ToClientCommand
 			u8[len] player name
 	*/
 
+	TOCLIENT_MODCHANNEL_MSG = 0x57,
+	/*
+		u16 channel name length
+	 	std::string channel name
+	 	u16 message length
+	 	std::string message
```

Comment:
i assume the documentation is still missing the changes that add a sender field

Reply:
done